.travis.yml

```diff
@@ -12,8 +12,10 @@ before_script:
   - sleep 3
 script:
   - cd vms
+  - flake8 cmdbox
```

Comment:
merge this with #480. Also you need fork this in your local, enable travis for your fork and see if it is up after your changes. Closing this one. I added comments there. Thanks!